### PR TITLE
test: update stale test expectations

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: tests/library/browsercontext-cookies.spec.ts [hunk 1]
```diff
@@ -353,7 +353,7 @@ it('should be able to send third party cookies via an iframe', async ({ browser,
 it('should support requestStorageAccess', async ({ page, server, channel, browserName, isMac, isLinux, isWindows }) => {
   it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/17285' });
   it.skip(browserName === 'chromium', 'requestStorageAccess API is not available in Chromium');
-  it.fixme(channel === 'firefox-beta', 'hasStorageAccess returns true, but no cookie is sent');
+  it.skip(channel === 'firefox-beta', 'hasStorageAccess returns true, but no cookie is sent');
```
Comment (yury-s) [inline tests/library/browsercontext-cookies.spec.ts:356]: is it still the case in current beta?

File: tests/library/browsercontext-page-event.spec.ts [hunk 2]
```diff
@@ -189,6 +189,7 @@ it('should not hang on ctrl-click during provisional load', async ({ context, pa
   it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/11595' });
   it.skip(browserName === 'chromium', 'Chromium does not dispatch renderer messages while navigation is provisional.');
   it.fixme(browserName === 'webkit' && isWindows, 'Timesout while trying to click');
+
```
Comment (yury-s) [inline tests/library/browsercontext-page-event.spec.ts:192]:
```suggestion
```

File: tests/library/download.spec.ts [hunk 3]
```diff
@@ -525,7 +525,7 @@ it.describe('download event', () => {
 
   it('should be able to cancel pending downloads', async ({ browser, server, browserName, browserVersion }) => {
     // The exact upstream change is in b449b5c, which still does not appear in the first few 91.* tags until 91.0.4437.0.
-    it.fixme(browserName === 'chromium' && Number(browserVersion.split('.')[0]) < 91, 'The upstream Browser.cancelDownload command is not available before Chrome 91');
+    it.skip(browserName === 'chromium' && Number(browserVersion.split('.')[0]) < 91, 'The upstream Browser.cancelDownload command is not available before Chrome 91');
```
Comment (yury-s) [inline tests/library/download.spec.ts:528]: can we remove it altogether or the electron is still on that version?

File: tests/library/download.spec.ts [hunk 4]
```diff
@@ -540,7 +540,7 @@ it.describe('download event', () => {
 
   it('should not fail explicitly to cancel a download even if that is already finished', async ({ browser, server, browserName, browserVersion }) => {
     // The exact upstream change is in b449b5c, which still does not appear in the first few 91.* tags until 91.0.4437.0.
-    it.fixme(browserName === 'chromium' && Number(browserVersion.split('.')[0]) < 91, 'The upstream Browser.cancelDownload command is not available before Chrome 91');
+    it.skip(browserName === 'chromium' && Number(browserVersion.split('.')[0]) < 91, 'The upstream Browser.cancelDownload command is not available before Chrome 91');
```
Comment (yury-s) [inline tests/library/download.spec.ts:543]: ditto

**Review by yury-s — APPROVED** [review]

File: tests/library/browsercontext-cookies.spec.ts (hunk 1 above)
Comment (dgozman) [reply tests/library/browsercontext-cookies.spec.ts:356]: It is still the case.

**Pull request merged**